docs/Kids-First/Setting-up-your-KF-Portal-Permissions/KF_1_PortalOverview.md

```diff
@@ -16,35 +16,39 @@ available to empower your research today. Data collected from more than
 
 Our aim is to teach researchers basic concepts, skills, and tools for
 working with data so that they can get more done in less time, and with
-less pain. The goal of this training module is to enable clinician
+less pain. **The goal of this training module is to enable clinician
 researchers to use the data portal to build cohorts, and to move that
-data to an analysis platform ([Cavatica](https://cavatica.squarespace.com/)). If you are looking for
+data to an analysis platform ([Cavatica](https://cavatica.squarespace.com/))**. 
+
+<!-- If you are looking for
 information on running workflows in Cavatica, please use \<lesson in
 progress\>.
+-->
 
 Please note that workshop materials are in "alpha" development.
 
-!!! Objectives
-    **In this lesson you will learn:**
+---
+
+ **In this lesson you will learn:**
 
-    -   How to connect your Kids First, Cavatica, and protected access
-    accounts
+    -   How to connect your Kids First, Cavatica, and protected access accounts
     -   How to find and filter Kids First data sets
     -   How to move your chosen cohort to Cavatica for analysis
     -   How to apply for access to new data sets
 
+---
 
-!!! Pre-requisites
-
-    **Background Knowledge**
+Pre-requisites
+--------------
+   **Background Knowledge**
 
-       This training assumes a basic background in human medicine
+   + This training assumes a basic background in human medicine
 
-    **Software**
+**Software**
 
-       This training will require a modern web browser
++ This training will require a modern web browser
 
-    **Credentials**
+**Credentials**
 
-       -   ORCID (preferred)
-       -   eRA Commons ID (preferred)
+   -   ORCID
```

Comment:
I'm not sure this is true...

Reply:
OK. I don't like this because its only true for parts of the tutorial. I think what I want is to make this 4 stand alone tutorials. Which just means adding a lesson start page (https://cfde-training-and-engagement.readthedocs-hosted.com/en/jeremyfixes/Kids-First/Setting-up-your-KF-Portal-Permissions/KF_1_PortalOverview/) for each section. That way, for the section on all the set up, you DO need an eRA and ORCID.

Reply:
These are super short tutorials... one or two steps... I'm not sure breaking them up further is effective (so I didn't). I was also not able to do tutorial steps without those two accounts so I'm not sure how someone else can (as in... if someone else has, please share!)  Preferred implies that someone can go through this without them... so perhaps we need to reword the prerequisite or put in a tip that says... hey... if nothing is working for you maybe it's because of you aren't using these. I think it's short enough and simple enough to do the job it's supposed to. 

I also approached this tutorial as a front to back, get accounts -> connect them -> acquire data -> move it to Cavatica -> analysis... though it is true that each one needs the previous step to move through the tutorial.  Any of the pieces alone don't have any context for me, but perhaps someone will come here looking for just... how to get an account on Cavatica?

Reply:
You can absolutely do the portal search stuff without ever knowing what Cavatica is, and with just a google account. The only thing you need Cavatica for is the push to cavatica part :) Also, I'm 99% sure that Abhijna has been working in Cavatica this whole time without an ORCID

Reply:
Yes! You can even download data from the KF portal and analyze it on your local computer (no need for Cavatica).

I used ERA commons to log into cavatica and my gmail to log into to KF. They talk to each other just fine. As of yesterday, I have all kinds of new data access too.

Reply:
The next part of the tutorial literally has CONNECT YOUR ORCID. So... at the very least there needs to be a lot of rewriting and organizing for it to make sense. This is why I deprecated the preferred. Is this why @amanda wants many tiny lessons broken out of this?

Reply:
I know that I am both awesome and brilliant, and therefore things I make are beyond reproach, but part of what I wanted ya'll to do is fix where my tutorial is bad. Like. You did what it says and decided that you need ORCID to do *those steps*...but...Do you even *need* all of those steps? Cavatica and the portal have been updated at least 3 times since I wrote this. It needs to be reality checked as much, if not more, than it needed to be grammar checked :)

Reply:
I made it a little bit shorter, I made sure it worked. Anything beyond that sounds more science. If the tutorial says get your ORCID and use it to do x and y... and then it works... I'm going to wrap myself in the sweet swaddling of success and make sure the words are spelled correctly. The tutorial got me doing all the things it said it was going to tell me to do. Generally, I think of that as 'awesome and brilliant' even if you want to come back and decide you were awful. Are we then getting rid of the ORCID steps altogether? Because you can't do ORCID steps without ORCID... still.

Reply:
Fair enough I guess. We're not getting rid of ORCID, but someone more sciencey needs to make sure we're not having people do pointless things. It can be me once this PR thing gets merged.